src/update-coords-names.txt

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#Rename x\1v( -> Xc<\1>(
+find ./ -name "*.*pp" -exec sed -i "s/\.x\([123]\)v(/.Xc<\1>(/g" {} \;
+
+#Rename x\1f( -> Xf<\1,\1>(
+find ./ -name "*.*pp" -exec sed -i "s/\.x\([123]\)f(/.Xf<\1,\1>(/g" {} \;
+
+#Rename x\1e( -> Xe<\1>(
+find ./ -name "*.*pp" -exec sed -i "s/\.x\([123]\)e(/.Xe<\1,\1>(/g" {} \;
+
+#Rename x\1s\2( -> Xs<\1,\2>(
+find ./ -name "*.*pp" -exec sed -i "s/\.x\([123]\)s\([123]\)(/.Xs<\1,\2>(/g" {} \;
+
+#Rename dx\1v -> Dxc<\1>)
+find ./ -name "*.*pp" -exec sed -i "s/\.dx\([123]\)v(/.Dxc<\1>(/g" {} \;
+
+#Rename dx\1f -> Dxf<\1,\1>
+find ./ -name "*.*pp" -exec sed -i "s/\.dx\([123]\)f(/.Dxf<\1,\1>(/g" {} \;
+
+#Rename Dx(, -> cellWidth(,
+find ./ -name "*.*pp" -exec sed -i "s/\.Dx(/.cellWidth(/g" {} \;
+
+#Rename Area(, -> faceArea(,
+find ./ -name "*.*pp" -exec sed -i "s/\.Area(/.faceArea(/g" {} \;
+
+#Rename Volume(, -> CellVolmue(,
+find ./ -name "*.*pp" -exec sed -i "s/\.Volume(/.CellVolume(/g" {} \;
```

Comment:
:+1: no wonder all the changes seemed correct when I looked closely lol

Reply:
Oh oops I didn't mean to add this file to the version control... but maybe it's worth leaving it here as a record of the change?

Reply:
Hm yeah maybe this file should be removed from the repo, since I'm not sure how likely it is that something very similar to this script will be needed again

Reply:
Removed. :+1: